app/controlplane/plugins/core/slack-webhook/v1/slack_webhook.go

```diff
@@ -0,0 +1,208 @@
+//
+// Copyright 2023 The Chainloop Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package slack_webhook
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io"
+	"net/http"
+	"strings"
+	"text/template"
+
+	"github.com/chainloop-dev/chainloop/app/controlplane/plugins/sdk/v1"
+	"github.com/go-kratos/kratos/v2/log"
+)
+
+type Integration struct {
+	*sdk.FanOutIntegration
+}
+
+// 1 - API schema definitions
+type registrationRequest struct {
+	WebhookURL string `json:"webhook" jsonschema:"format=uri,description=URL of the slack webhook"`
+}
+
+type attachmentRequest struct{}
+
+// 2 - Configuration state
+type registrationState struct{}
```

Comment:
I am assuming that there is no way to retrieve any data from the webhook similarly to what we did with Discord right?

Also, if `registrationState` has no state, you might don't need to neither create this struct nor return it.

Reply:
Nothing there:

```
$ curl -v -X POST -H 'Content-type: application/json' --data '{"text":"Hello!"}' https://hooks.slack.com/services/T0...R
[...]
< HTTP/2 200
< date: Tue, 04 Jul 2023 22:03:03 GMT
< server: Apache
< vary: Accept-Encoding
< strict-transport-security: max-age=31536000; includeSubDomains; preload
< referrer-policy: no-referrer
< access-control-allow-origin: *
< x-frame-options: SAMEORIGIN
< content-type: text/html
< content-length: 2
< via: 1.1 slack-prod.tinyspeck.com, envoy-www-iad-gkzjvaou, envoy-edge-fra-zfgbdwqn
< x-envoy-upstream-service-time: 148
< x-backend: main_normal main_canary_with_overflow main_control_with_overflow
< x-server: slack-www-hhvm-main-iad-fafg
< x-slack-shared-secret-outcome: no-match
< x-edge-backend: envoy-www
< x-slack-edge-shared-secret-outcome: no-match
<
* Connection #0 to host hooks.slack.com left intact
ok%
```